src/app/atoms/CopyButton.tsx

```diff
@@ -96,11 +98,12 @@ const CopyButton: FC<CopyButtonProps> = ({
         }
         {...rest}
         onClick={handleCopyPress}
+        {...setTestID(ExploreSelectors.valueButton)}
```

Comment:
you could not use specific `ExploreSelectors...` for shared component - pass them as props.  
Additionally, weird naming `valueButton`